resources/js/graphql/mutation/auth/UpdateUser.ts

```diff
@@ -1,3 +1,3 @@
-export default `mutation UpdateUser($email: String, $password: String) {
-    UpdateUser(email: $email, password: $password)
+export default `mutation UpdateUser($email: String, $password: String, $account: String) {
+    UpdateUser(email: $email, password: $password, account: $account)
```

Comment:
This attribute is dedicated for wallet daemon and we cannot update this. I created a new attribute for this in this PR https://github.com/enjin/platform-multi-tenant/pull/72

Reply:
okay thank you! will check the new implementation